### PR TITLE
Dashboard Updated

### PR DESCRIPTION
**Pull request by p-harsh**
Screenshot (32)
Screenshot (28)
Screenshot (29)
Screenshot (30)
Screenshot (33)

### CODE REVIEW

File: src/views/Index.js [hunk 1]
```diff
@@ -15,322 +15,182 @@
...
+                <FormGroup check disabled>
+                  <Label check>
+                    <Input type="radio" name="radio1" disabled />{' '} Option three is disabled</Label>
+                </FormGroup>
+              </FormGroup>
+              <FormGroup check>
+                <Label check>
+                  <Input type="checkbox" />{' '} Check me out</Label>
+              </FormGroup>
+            </Form>
```
Comment (kartikcode) [inline src/views/Index.js]: Implement a generic component for the form wherein the `FormGroup` would be mapped from a data object. You can keep all required fields as textbox. Also

File: src/views/Index.js [hunk 2]
```diff
@@ -15,322 +15,182 @@
...
+      .catch(error => console.log(error))
+  }, [])
+
+  const toggle = () => setIsModal(!isModal);
+
+  const renderRowSubComponent = (fetchedData, cells) => {
+    let index = parseInt(cells[0]['row']['id'])
+    let headingstyle = { fontSize: '20px' }
+    return (
+      <Card style={{ margin: "0 auto", color: 'white', background: 'linear-gradient(87deg, #11cdef, #1171ef)' }}>
```
Comment (kartikcode) [inline src/views/Index.js]: Maybe improve the background color. White would look descent enough. And try and match the font styles.

**Comment by kartikcode** [comment]
Great Job!!
I've added some code reviews. Check them out once. Also, I think it would be better if you could use Argon's modal component for creating the form. If not, then create a separate `FormModal` component. A generic component for the same would be better.

**Comment by kartikcode** [comment]
Kindly post in the screenshots for the updated UI.

**Comment by p-harsh** [comment]
Screenshot (40)
Screenshot (41)
Screenshot (42)
Screenshot (43)
Screenshot (45)

**Pull request merged**